tests/kernel/boot_ltp.pm

```diff
@@ -147,8 +151,14 @@ EOF
         script_run('cat /etc/hosts');
 
         script_run('ip addr');
+        script_run('ip netns exec ltp_ns ip addr');
         script_run('ip route');
         script_run('ip -6 route');
+
+        script_run('ping -c 2 $IPV4_NETWORK.$LHOST_IPV4_HOST');
```

Comment:
Why not to use assert_script_run in all of those calls? We will still be able to see the output, but in case of issues there will be no chance to miss it.

Reply:
@rwx788 These are informative. I don't want to fail testing just because there is something broken with them, that would be false positive (many tests don't need these variables).